OutOfSchool/OutOfSchool.WebApi/Models/ModelsDto/DirectionOfEducationDTO.cs

```diff
@@ -0,0 +1,11 @@
+﻿using System.Collections.Generic;
+
+namespace OutOfSchool.WebApi.Models.ModelsDto
+{
+    public class DirectionOfEducationDTO
```

Comment:
can it be something shorter, like 'Category'?

Reply:
DirectionOfEducationDTO ->> CategoryDTO
ProfileOfEducationDTO ->> SubcategoryDTO

Can it be so?

Reply:
sounds good